usaspending_api/settings.py

```diff
@@ -227,6 +227,63 @@
     },
 }
 
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'default-loc-mem-cache',
+    },
+    'locations': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'locations-loc-mem-cache',
+    },
+}
+
+# Cache environment - 'local', 'disabled', or 'elasticache'
+CACHE_ENVIRONMENT = 'local'
+
+# Set up the appropriate elasticache for our environment
+CACHE_ENVIRONMENTS = {
+    # Elasticache settings are changed during deployment, or can be set manually
+    'elasticache': {
+        'BACKEND': 'django_elasticache.memcached.ElastiCache',
+        'LOCATION': 'ELASTICACHE-CONNECTION-STRING',
```

Comment:
These strings are overriden by the deployment script. This allows us to _not_ store AWS strings in our repo.